app/knights/preparation.py

```diff
@@ -0,0 +1,19 @@
+class Knight:
+    def __init__(self, knight_and_amo: dict):
+        self.name = knight_and_amo["name"]
+        self.power = knight_and_amo["power"]
+        self.hp = knight_and_amo["hp"]
+        self.dr = 0

```

Comment:
I think dr = defense rate but I've been playing video games all my life. Bad name for protection

Reply:
it's not from computer games it's from tabletop RPG games GURPS, but ep, you are right